configs/demo.json

```diff
@@ -1,14 +1,12 @@
 {
   "backend": {
-    "artifactory": {
-      "master": "https://artifactory.host",
-      "slave": "https://artifactory.host",
-      "release": "DEMO-release",
-      "vgroup": "DEMO-group",
-      "credentials": null
-    }
+    "local": {}
   },
   "environments": {
+    "alp36": {
```

Comment:
"alpine36" or "alpine".
Is there a necessity for this specific tag? What do we do about upgrades?

Reply:
I'll expand it, sure.

Not entirely sure if we need to be this specific either. It was intended as an abi safety by tag, but as long as the app at the top is at least newer, then we should be good. At least that was the case for ubuntu's. Alpine doesn't use glibc I guess, so maybe we can just use libraries across different alpines in newer and older alpine containers.